### PR TITLE
Only output report if there is something to report.

### LINKED ISSUE #104: Suppress the report entirely if there are no flaky tests
`--no-success-flaky-report` still outputs an empty flaky report. It'd be nice if it suppressed the output altogether. I'm happy to create a PR for this if you're willing to accept it.

### PR DESCRIPTION
**Pull request by jonparrott**
Fixes #104

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @jonparrott, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: flaky/_flaky_plugin.py [hunk 1]
```diff
@@ -377,13 +377,19 @@ def _add_flaky_report(self, stream):
         :type stream:
             `file`
         """
+        value = self._stream.getvalue()
+
+        # If everything succeeded and --no-succes-flaky-report is specified
```
Comment (Jeff-Meadows) [inline flaky/_flaky_plugin.py]: typo: --no-succes**s**-flaky-report

Comment (jonparrott) [inline flaky/_flaky_plugin.py]: Fixed. :)

**Comment by jonparrott** [comment]
@Jeff-Meadows @boxcla I'm waiting for our Open Source lawyers to approve me signing the CLA. I'll get that resolves as soon as I can. Thanks!

**Comment by jonparrott** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @jonparrott has just signed the CLA. Thanks, and we look forward to your contribution.

**Comment by jonparrott** [comment]
Thanks! When can I expect the next release?

**Comment by Jeff-Meadows** [comment]
I just pushed a new release. flaky 3.2.0 has your changes!

On Wed, Jul 20, 2016 at 4:41 PM, Jon Wayne Parrott <notifications@github.com

**Comment by jonparrott** [comment]
Thank you much!

**Pull request merged**